File: FrontEnd/src/app/core/models/Media/Media.ts
export interface MediaQueryDto {
  searchTerm?: string;
}

export interface Media {
  id: number;
  title: string;
  description: string;
  createdAt: Date;
  updatedAt: Date;
  mediaTypeId: number;
}

export interface CreateMediaDto {
  title: string;
  description: string;
  mediaTypeId: number;
}

File: BackEnd/Models/Media/Media.cs
namespace Models;

public class Media
{
    public int Id { get; set; }
    public required string Title { get; set; }
    public string? Description { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }

    public required int MediaTypeId { get; set; }

    public virtual MediaType? MediaType { get; set; }
}
